app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -17,8 +17,6 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     end
 
     api :GET, "/organizations/:organization_id/subscriptions", N_("List organization subscriptions")
-    api :GET, "/systems/:system_id/subscriptions", N_("List a content host's subscriptions"), :deprecated => true
-    api :GET, "/activation_keys/:activation_key_id/subscriptions", N_("List an activation key's subscriptions")
```

Comment:
How do the changes being made to system for scoped search conversion play into this? Do they change the workflow of how this data is collected at all? I know we've chatted about this prior, but it's all a big jumble in my head.